B23 Ex03 Ronen 319047718 Ido 315942193/VehicleFactory.cs

```diff
@@ -148,8 +243,20 @@ private bool getTransportingHazardousMaterial()
 
         private eCarColor getColorFromUser()
         {
-            Console.WriteLine("Please insert the bike license type\n1. White\n2. Black\n3. Yellow\n4. Red");
-            Enum.TryParse(Console.ReadLine(), out eCarColor o_carColor);
+            Display.ColorMenu();
+
+            if (Enum.TryParse(Console.ReadLine(), out eCarColor o_carColor))
```

Comment:
remove o_, this is only done in input parameters in functions

Reply:
ok 
but there are a lot of this in the code

Reply:
i will do it in other bracnh
fix all